test/protocol/ProtocolInitializationHandlerTest.js

```diff
@@ -655,32 +658,100 @@ describe("ProtocolInitializationHandler", async function () {
     let facetCut;
     let calldataProtocolInitialization;
     let minResolutionPeriod;
+    let snapshotId;
+    let protocolDiamondAddress;
 
     beforeEach(async function () {
-      version = "2.2.1";
-
-      const facetsToDeploy = await getV2_2_0DeployConfig(); // To deploy 2.2.1, we can use 2.2.0 config
+      if (snapshotId) {
+        await revertToSnapshot(snapshotId);
+        snapshotId = await getSnapshot();
+      } else {
+        version = "2.2.1";
+        protocolDiamondAddress = await protocolDiamond.getAddress();
+
+        // NEED TO ACTUALLY DEPLOY VOUCHER IMPLEMENTATIONS
+        const protocolClientArgs = [protocolDiamondAddress];
+        const [, beacons] = await deployProtocolClients(
+          protocolClientArgs,
+          maxPriorityFeePerGas,
+          [rando.address] // random address in place of forwarder
+        );
+        const [beacon] = beacons;
+
+        const facetsToDeploy = await getV2_2_0DeployConfig(); // To deploy 2.2.1, we can use 2.2.0 config
```

Comment:
I prefer to set this in the `00_config.js:getFacets` function as it is the designated location for storing all version configurations (which can then be reused for upgrade tests). but I am willing to approve this way as I can change it when I begin working on https://github.com/bosonprotocol/boson-protocol-contracts/pull/680.

Reply:
do you want to add a note so we don't lose this one ? @anajuliabit

Reply:
done